hnn_core/tests/test_params.py

```diff
@@ -76,57 +74,6 @@ def test_base_params():
     assert params == params_base
 
 
-def test_convert_to_hdf5(tmp_path):
```

Comment:
Yep! Actually I have a subsequent PR ready for the function to convert flat json and param files to the hierarchical json format. It has these tests converted for that function. These just needed to be deleted here because this PR deletes the read_network function.